navigator.lua

```diff
@@ -4,9 +4,19 @@
 -- Contributors: SteveJobzniak
 -- URL: https://github.com/donmaiq/mpv-filenavigator
 --
+
+local utils = require 'mp.utils'
+
+ON_WINDOWS = (package.config:sub(1,1) ~= '/')
```

Comment:
This seems to work on Linux and Windows but out of curiosity do you know if this package config is documented anywhere?

Reply:
Was looking how other scripts handle OS detection and I just copypasted it. https://www.lua.org/manual/5.2/manual.html#pdf-package.config seems to be the official explanation of what it does.